plugins/filetools/plugin.js

```diff
@@ -104,14 +104,10 @@
 						data.message = response.error.message;
 					}
 
-					// But !uploaded means error.
-					if ( !response.uploaded ) {
-						evt.cancel();
```

Comment:
Is it really necessary to not cancel the event? Tests are still passing with `evt.cancel`.